virtualhost.sh

```diff
@@ -72,7 +72,7 @@ if [ "$action" == 'create' ]
 		<VirtualHost *:80>
 			ServerAdmin $email
 			ServerName $domain
-			ServerAlias $domain
+			ServerAlias www.$domain
```

Comment:
Hi, your comment is related to a previous commit. In the next commits [9b6f0f0](https://github.com/RoverWire/virtualhost/pull/21/commits/9b6f0f0f12b1414737bb01cac4c11a236261030d) and [240549d](https://github.com/RoverWire/virtualhost/pull/21/commits/240549d76048b497962d713bbb36e177c8e43e04) I changed the behavior as you requested, and moreover added the possibility to create a custom alias (not only www.site.com but whateveryouwant.another.site ) updating the documentation